### PR TITLE
goad 1.4.1 (new formula)

### PR DESCRIPTION
**Pull request by gserra-olx**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/goad.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Goad < Formula
+  desc "AWS Lambda powered, highly distributed, load testing tool built in Go."
+  homepage "https://goad.io/"
+  url "https://github.com/goadapp/goad.git"
```
Comment (bfontaine) [inline Formula/goad.rb]: We need a tag here. You can should be able to use the tarball at https://github.com/goadapp/goad/archive/v1.4.0.tar.gz and remove the `version` below; Homebrew will guess it from the URL.

File: Formula/goad.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  version "1.4.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath/"bin"
+    ENV.prepend_create_path "PATH", buildpath/"bin"
```
Comment (bfontaine) [inline Formula/goad.rb]: Do we really need this?

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/goad.rb (hunk 2 above)
Comment (cwaltken-edrans) [inline Formula/goad.rb]: Right now this line is not obsolete because we rely on `go-bindata` in the build process.

**Comment by gserra-olx** [comment]
@bfontaine , @cwaltken-edrans  applied all the changes you requested, I just forgot to tag our repo

is there a way to re-kick the build process?

**Comment by gserra-olx** [comment]
@bfontaine seems we made it. Can u have a look, please?

**Review by bfontaine — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @gserra-olx for your contribution to Homebrew!

**Comment by gserra-olx** [comment]
our pleasure @fxcoudert ;)

**Pull request closed without merging**